FakeXrmEasy.Shared/FiscalYearSettings.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Xrm.Sdk;
+using Microsoft.Xrm.Sdk.Client;
+using System;
+
+namespace FakeXrmEasy
+{
+    [EntityLogicalName("organization")]
```

Comment:
Thanks a mill for this @npavlov ! Curious to know where this EntityLogicalName was generated from...  crmsvcutil or XrmToolkit or similar?